src/gtest/pastel_gtest_main.cpp

```diff
@@ -26,6 +28,7 @@ struct ECCryptoClosure
 
 ECCryptoClosure instance_of_eccryptoclosure;
 CPastelTest_Environment *gl_pPastelTestEnv = nullptr;
+fs::path pathTemp;
```

Comment:
don't use global variable here, instead you can use the one that I have already in CPastelTest_Environment class:
m_TempDataDir.
This is used in InitializeChainTest/FinalizeChainTest and cleaned up in FinalizeChainTest via ClearTempDataDir().
I think you were moving this pathTemp from test_bitcoin.cpp, right?

Reply:
done thanks

Reply:
previously, there is no InitializeChainTest/FinalizeChain in the pastel_gtest_main. Then I tried to setup one. Later you have done in the recent PR. Then I tried to merge master with your changes and still not removing all my changes in pastel_gtest_main. Fix Thanks